### ISSUE TITLE
Methman Monument too easy to break

### ISSUE THREAD

**Issue by SoloAceMouse** [jasonrohrer/OneLifeData7#1184]
![Meth monument issue](https://github.com/jasonrohrer/OneLifeData7/assets/147592460/e1018c67-09c2-442c-b642-06f4c3bef154)

Currently, the item "Additional Methman Monument" requires a minimum of 15 hours to construct and can be destroyed in under a second by anyone carrying a pickaxe.

I propose that the monument be changed so that a two-person elder receipt system is necessary for removal. I believe doing so would discourage the majority of low-effort vandalism attempts by players wishing to deface the monument.

Thank you for adding the monument, btw, it is a nice tribute.

**Comment by SoloAceMouse** [comment]
Correct, however a fence requires regular maintenance or it decays.

By adding elder receipt protection to the monument it can more easily be constructed centrally, where people can visit it and not have to hide it inside a property fence enclosure.

**Comment by DougLefelhocz** [comment]
I agree with elder receipt protection.  It also symbolically works, since he was an elder, having played since launch, and in years to many of us.

But, I dare suggest going even further.

Either:

1. An elder removal receipt followed by a swing of an axe, since he cut so many maples in the early rift.  

Or

2.  An elder removal receipt followed by the use of a sword, since as Eve Tarr X, Eve Tarr XXII, and Eve Tarr XXXVIII [he used a sword many, many times](https://discord.com/channels/328215300279500800/328215300279500800/817058277233393764) (I don't find him calling himself Eve Tarr, but I suspect that he did use that name).

That way, one could imagine that at least the spirit of the old man himself approves of the destruction.

**Comment by SoloAceMouse** [comment]
Frankly, we know this community.

People will try to break it. OHOL is a game utterly rife with people who spend all day trying to destroy everything they can.  The only saving grace is that most of them are lazy and low effort.

If you let them destroy a monument to a dead, respected member of the community with a single pickaxe strike then nobody is ever going to build one of these monuments.

![image](https://github.com/jasonrohrer/OneLifeData7/assets/147592460/3fb92691-208b-4309-8b3a-d9286ed7a651)

The journey to 0,0 is an interesting proposal, but why bother if some jackass is gonna grief it the moment the thing gets built.

Please add elder receipt protection [or something] to the memorial.

**Comment by DopiePanda** [comment]
You should be able to make a statue template from an already copied statue. Travel time now on horse to get to the monument is roughly 16 hours, each direction, and this is increasing each day.

So spending 32 hours total to get the template back for someone to be able to break it within minutes is why no one has gone yet.

I think it is a fair trade-off, cause if at any point the statues are not being copied, you'd have to go all the way back to 0,0 to get a new template again.

And even if planes are unlimited travel distance, someone has to go and make the landing strip there that only lasts for 2 hours.

**Comment by DougLefelhocz** [comment]
> You should be able to make a statue template from an already copied statue. 

The steam advertisement says:

>  Leave a legacy for the next generation as you help to rebuild civilization from scratch. 

The Original Methman Monument sitting at the origin can get thought of as at or near "scratch".  An Additional Methman Monument can get thought of as rebuilding such a monument.  Consequently, needing to go back to the original statue fits the theme of rebuilding civilization from scratch.  No particular object in the game that I know of fits that the concept of "rebuilding from scratch" so well.  If the we could get plans from an Additional Methman Monument, anything made from those plans would not have gotten rebuilt from scratch.  So, the concept ends up weakened, if not nullified also, were we able to use Additional Methman Monuments.

> ... So spending 32 hours total 

Rather than theoretical estimates, I prefer MrXIX's estimate of 20k an hour, which he did with a truck, getting [100k out](https://discord.com/channels/328215300279500800/328215300279500800/1205538516424065115) from his leader in [5 hours](https://discord.com/channels/328215300279500800/328215300279500800/1205542428501868584).  580 k tiles at 20k an hour makes for 29 hours of travel one way.

Also, the game development philosophy is "everything runs out".  Using copied statues does not fit that well, because making more statutes does not run out.  But, with needing to go back to the origin, eventually making Additional Methman Monuments run out.  Then it would make sense to go back to scratch (via an apocalypse or Jason doing a hard wipe or making a cron-job for resetting the server).

**Comment by DopiePanda** [comment]
Yes they might have been made on the small servers, but only a fraction of players play on other servers than BS2. Most don't even know there are other servers.

And no Methman monuments have been made on BS2 yet.

**Comment by DougLefelhocz** [comment]
> And no Methman monuments have been made on BS2 yet.

Because there was no plan for how such would happen on bs2.  Nor any plan for how such would happen on s1.

**Comment by DopiePanda** [comment]
Well. There is only one way to do it, start heading east for over a day in a row, non-interrupted.

There is no other way.

Once you get there you can set up an airstrip and have someone fly out to get the templates (flight limit update not pushed to live yet tho)

But if you dont go back and make a new airstrip every 2 hours (for the rest of time), you'll have to do the 600k trip all over again

**Comment by DougLefelhocz** [comment]
> Well. There is only one way to do it, start heading east for over a day in a row, non-interrupted.
> 
> There is no other way.
> 
> Once you get there you can set up an airstrip and have someone fly out to get the templates (flight limit update not pushed to live yet tho)
> 
> But if you dont go back and make a new airstrip every 2 hours (for the rest of time), you'll have to do the 600k trip all over again

That isn't what I did on s5.  I don't think it's what SamWish did on some other server.

Also, I don't believe that there exists any planes on server 1.  Nor any kerosene.

If Eve positions were reset to their original area on bs2 also, going east for hours would not be necessary to get plans.

Lastly, _as I write this_, max plane distance is 10,000 tiles (and it has been that way since the Original Methman Monument got put on the map).  If someone were at the origin, no plane could fly there in one trip from where families in the main play area are right now.

**Comment by ShamWish** [comment]
> That isn't what I did on s5. I don't think it's what SamWish did on some other server.

The server I did it on, the statue was WAY closer than it is to the families of BS2. 26K tiles is hardly a feat. Dopie was clearly taking about BS2 in the post you quoted, Doug. Comparing the time it would take on that server to other servers is pretty silly. 

I would also prefer being able to make the plans from a replica statue. It would feel a lot more like carrying on Methman's memory from town to town. 

Having to go several hundreds of thousands of tiles every time you run out of plans is very lame. I doubt anyone will keep up on making statues, if anyone ever does make it to the origin. <- ALL THIS IS IN REFRENCE TO BS2 BTW

**Comment by DougLefelhocz** [comment]
> The server I did it on, the statue was WAY closer than it is to the families of BS2. 26K tiles is hardly a feat. Dopie was clearly taking about BS2 in the post you quoted, Doug. Comparing the time it would take on that server to other servers is pretty silly.

I disagree on the time comparisons as silly.  Such comparisons suggest something about the quality of Eve spawns and family positions on BS2 compared to other servers.  Since the Original Methman Monument *and* the Tarr Monument take so much time to reach on bs2 compared to servers 2 through 15, bs2 has lower quality of Eve spawns and family areas than servers 2 through 15.  Servers 2 through 15 have better quality Eve spawns (for Eves not on an Eve chain) and family areas.

> Having to go several hundreds of thousands of tiles every time you run out of plans is very lame. I doubt anyone will keep up on making statues, if anyone ever does make it to the origin.

Though I disagree on plans coming from Additional Methman Monument's (since then making another statue would not qualify as getting "rebuilt from scratch"), I do agree on each point I quoted here.

If we regularly had arc resets before 100,000 tiles got reached by families (which at 1000 tiles a day would approximately equal 100 days at most) which were hard wipes that pushed Eves back to around the origin, it would not be so lame.  And people would make more Additional Methman Monument statues.  And even if people re-settled in advanced towns, sometimes, they might still have the opportunity to journey back to near (0, 0) without a large and boring timesink, and help to rebuild part of civilization from scratch by making an Additional Methman Monument by getting plans from the Original Methman Monument.

**Comment by DougLefelhocz** [comment]
The person who was Eve of Swift family mentions An Additionally Methman Monument getting [broken](https://discord.com/channels/328215300279500800/328215300279500800/1269259519246995460).
